src/components/form/button.js

```diff
@@ -98,6 +98,7 @@ export const LinkButton = styled(LocalizedLink)`
     ${SharedButtonStyle}
     text-align: center;
     text-decoration: none;
+    display: inline-block;
```

Comment:
any reason why?, im afraid this will break all buttons

Reply:
If I'm not mistaken when it's not `inline-block`, the element `<a>` does not cover the wraper around it and it makes problem for margins of other elements, since the margin will be calculated from the `<a>` link, not the whole wrapper. I checked few forms and it was ok, instead of `404 page` which has a little difference in margin and I can fix it on this PR.